### ISSUE TITLE
Django 1.4 Support

### ISSUE THREAD

**Issue by frnhr** [runekaagaard/django-admin-filtrate#6]
Django 1.4 brings some changes to admin filters, so Filtrate filter will brake. Loudly.

http://yuji.wordpress.com/2011/06/15/django-1-4-alpha-custom-list-filter-rip-filterspec/

**Comment by mvattuone** [comment]
This sort of works, but I had to do a few things to get it to work using 1.7.1, rather than having immediate out of the box support.  It still saved me a ton of time though, so thank you!  I just wanted to pass off some notes for you, hopefully useful...

- I had to override the is_active function such that it was only looking for the filtrate class.  I think maybe it is because I'm using a bootstrap admin package, but I did not have #changelist-filter in my DOM.
- In your test for the 1.4 version, you have a list filter that contains a tuple of the parameter name and the class name.  It should just be the class name.  
- If I click on the datepicker, it closes the dropdown behind it, which is sort of poor UX.  Not sure if that's something at the datepicker level, but I may look into it some more if I can find some time.  At the very least, filtering by date is now possible!
